Formula/zsh-async.rb

```diff
@@ -0,0 +1,28 @@
+class ZshAsync < Formula
+  desc "Perform tasks asynchronously without external tools"
+  homepage "https://github.com/mafredri/zsh-async"
+  url "https://github.com/mafredri/zsh-async/archive/v1.8.5.tar.gz"
+  sha256 "3ba4cbc6f560bf941fe80bee45754317dcc444f5f6114a7ebd40ca04eb20910a"
+  license "MIT"
+
+  bottle :unneeded
+
+  def install
+    cp "async.zsh", "async"
+    zsh_function.install "async"
+    prefix.install Dir["*.zsh"]
```

Comment:
This will install four additional files — `async.plugin.zsh`, `async.zsh`, `async_test.zsh`, and `test.zsh`, into the root of the keg solely to be used by the `test do` block.

Running the package's full test suite might be a little bit overkill in this case, particularly when it does not come for free, instead, every user will need to install these four additional files.

For the `test do` block, I would suggest to run the package's own [example code](https://github.com/mafredri/zsh-async#example-code) and then verify the output.

Reply:
@lembacon Done.